operator/doublewrite/metrics.go

```diff
@@ -10,46 +10,46 @@ import (
 
 func NewMetrics() *Metrics {
 	return &Metrics{
-		IdentityCRDTotal: metric.NewGauge(metric.GaugeOpts{
+		IdentityCRD: metric.NewGauge(metric.GaugeOpts{
 			Namespace: metrics.CiliumOperatorNamespace,
-			Name:      "doublewrite_identity_crd_total",
+			Name:      "doublewrite_identity_crd",
```

Comment:
Is this a test commit? I'd expect to see the suffix `_total` (or other units) to pass promtool checks.

Reply:
Oh I see https://github.com/cilium/cilium/pull/37079#issuecomment-2601773770 .

Reply:
`doublewrite_identity_crd` doesn't make sense to me though, there's no units. What does it mean for there to be a number of doublewrite identity crd?

Reply:
Do these metrics need to be Gauges?

Reply:
Also CRD is Custom Resource Definition, so the "template" if you will that we install into kube-apiserver to allow users to configure Custom Resources. Maybe the terse name is not quite capturing the correct terminology?

Reply:
I realize this PR is just trying to make the linter happy, but these are the questions that occur to a @cilium/metrics reviewer :thinking:

Reply:
> It looks to me like the actual unit here for all of these metrics is identities, so that should be the suffix.

Yeah I think the name itself has the subsystem in the spot of the metric. This should probably be `doublewrite_kvstore_identities` instead. Same applies to the rest.

Reply:
I just made a separate PR for this: https://github.com/cilium/cilium/pull/37248

Reply:
ah sorry, I actually pushed the commit to address what was discussed here as well. However, I am happy to go any direction. 

Thanks a lot for your contribution.

Reply:
ah my bad, I missed it, I only saw your comment about tackling this in a separate PR so I opened one 😄 I am fine with anything, we can also merge the changes as part of your PR